src/nuspec/chocolatey/docfx/docfx.nuspec

```diff
@@ -72,6 +72,7 @@ If you are interested in creating your own website with your own styles, you can
   <files>
     <!-- this section controls what actually gets packaged into the Chocolatey package -->
     <file src="tools\**" target="tools" />
+    <file src="..\..\..\..\LICENSE" target="tools\" />

```

Comment:
Another way to include this file is to copy it explicitly. We already have similar logic here for `build` and `content` folder: https://github.com/dotnet/docfx/blob/054d4f6a834dd7b6cfe6b609f5938531095876d6/pack.ps1#L74-L75